Singularity/Singularity/Game1.cs

```diff
@@ -114,6 +122,7 @@ protected override void LoadContent()
             mGameScreen.AddObject(mRoad1);
             mGameScreen.AddObject(road2);
             mGameScreen.AddObject(road3);
+            mGameScreen.AddObject(fow);
```

Comment:
These will eventually have to be moved to within the gamescreen itself to allow loading and stuff to work properly